Updating company-cadence.md (#115)
mattermost/mattermost-handbook: operations/operations/company-cadence.md

Every 6 months we'll be asking staff to spend around 3 minutes completing a staff enablement survey of 12 engagement questions, plus identifying their organizational leader. ‌

> * Enablement Survey will be announced in COM meeting 1-week before the survey goes live

Reviewer: ```suggestion
* Enablement Survey will be announced in COM meeting one week before the survey goes live.
```

> * Survey link will be shared at the beginning of COM 

Reviewer: ```suggestion
* The survey link will be shared at the beginning of COM.
```

> * Link is posted in COM channel with "Thumbs Up" emoji reaction

Reviewer: ```suggestion
* The link is posted in COM channel with "Thumbs Up" emoji reaction.
```

> * All staff are asked to complete the survey and signal completion by clicking on Thumbs Up

Reviewer: ```suggestion
* All staff are asked to complete the survey and signal completion by clicking on "Thumbs Up".
```

> * Ideally, the meeting will conclude after Thumbs Up count reaches attendee count but this is not a requirement. 

Reviewer: ```suggestion
* Ideally, the meeting will conclude after "Thumbs Up" count reaches attendee count but this is not a requirement. 
```

> * An analysis of the results should be prepared and presented in the next 2-4 weeks in COM

Reviewer: ```suggestion
* An analysis of the results should be prepared and presented within the next 2 - 4 weeks in COM.
```

Every 6 months we'll be asking staff to spend around 3 minutes completing a staff enablement survey of 12 engagement questions, plus identifying their organizational leader. ‌

> * Enablement Survey will be announced in COM meeting one week before the survey goes live.

Reviewer: ```suggestion
* The enablement survey will be announced in COM one week before the survey goes live.
```

* The survey link will be shared at the beginning of COM.

> * The link is posted in COM channel with "Thumbs Up" emoji reaction.

Reviewer: ```suggestion
* The link will be posted in the COM channel with "Thumbs Up" emoji reaction.
```
